tests/plugins/test_export_operator_manifests.py

```diff
@@ -0,0 +1,108 @@
+    data = dedent("""\
+        FROM foo
+        LABEL com.redhat.delivery.appregistry=%s
+        CMD foobar
+    """ % str(label).lower())
+    if not has_label:
+        data = dedent("""\
+            FROM foo
+            CMD foobar
+        """)
+    tmpdir.join('Dockerfile').write(data)
+
+
+def mock_workflow(tmpdir, image_path):
+    workflow = DockerBuildWorkflow(SOURCE, "mock:default_built")
+    workflow.source = StubSource()
+    builder = StubInsideBuilder().for_workflow(workflow)
+    builder.set_df_path(str(tmpdir))
+    builder.tasker = flexmock()
+    workflow.builder = flexmock(builder)
+    workflow.exported_image_sequence.append({'path': image_path})
+    return workflow
+
+
+def generate_image(tmpdir, image_path):
+    layer_dir = os.path.join(str(tmpdir), 'layerdir')
+    os.mkdir(layer_dir)
+    layer_path = os.path.join(layer_dir, 'layer.tar')
+    layer_tar = tarfile.open(layer_path, 'w')
+    manifests_dir = os.path.join(str(tmpdir), 'manifests')
+    os.mkdir(manifests_dir)
+    open(os.path.join(manifests_dir, 'stub.json'), 'w').close()
+    layer_tar.add(manifests_dir, arcname='manifests')
+    layer_tar.close()
+    data = '[{"Layers": ["layerdir/layer.tar"]}]'
+    manifest = tarfile.TarInfo('manifest.json')
+    manifest.size = len(data)
+    tar = tarfile.open(image_path, 'w')
+    tar.addfile(manifest, StringIO(data))
+    tar.add(layer_dir, arcname='layerdir')
+    tar.close()
+    shutil.rmtree(layer_dir)
+    shutil.rmtree(manifests_dir)
+    return tar
+
+
+def mock_env(tmpdir, docker_tasker, has_label=True, label=True, scratch=False, orchestrator=False):
+    image_path = os.path.join(str(tmpdir), 'img.tar')
+    generate_image(tmpdir, image_path)
+    build_json = {'metadata': {'labels': {'scratch': scratch}}}
+    flexmock(util).should_receive('get_build_json').and_return(build_json)
+    mock_dockerfile(tmpdir, has_label, label)
+    workflow = mock_workflow(tmpdir, image_path)
+    workflow.prebuild_results[PLUGIN_CHECK_AND_SET_PLATFORMS_KEY] = orchestrator
+    runner = PostBuildPluginsRunner(docker_tasker, workflow,
+                                    [{'name': ExportOperatorManifestsPlugin.key}])
+    return runner
+
+
+class TestExportOperatorManifests(object):
+    @pytest.mark.parametrize('scratch', [True, False])
+    @pytest.mark.parametrize('has_label', [True, False])
+    @pytest.mark.parametrize('label', [True, False])
+    @pytest.mark.parametrize('orchestrator', [True, False])
+    @pytest.mark.parametrize('selected_platform', [True, False])
+    def test_skip(self, docker_tasker, tmpdir, caplog, scratch, has_label,
+                  label, orchestrator, selected_platform):
+        runner = mock_env(tmpdir, docker_tasker, has_label=has_label, label=label,
+                          scratch=scratch, orchestrator=orchestrator)
+        result = runner.run()
+        if any([not has_label, not label, scratch, orchestrator, not selected_platform]):
```

Comment:
Don't need `[`/`]` here, `any(not has_label, not label, ...)` (it's implicitly a generator expression)

Reply:
It raises a TypeError in this case
```
any(iterable) -> bool
```

Reply:
Oh, I'm thinking of `any([... for ... in ...])` -> `any(... for ... in ...)`. Sorry for the noise.